spec/mail/fields/references_field_spec.rb

```diff
@@ -70,4 +70,9 @@
     m = Mail::ReferencesField.new( '2a26f8f146e27159@domain.com@domain.com 4769770500E92399@n064.sc1.he.tucows.com' )
     expect(m.message_ids).to eq [ '2a26f8f146e27159@domain.com@domain.com', '4769770500E92399@n064.sc1.he.tucows.com' ]
   end
+
+  it 'should be able to parse new lines' do
```

Comment:
checkout [line 40](https://github.com/getoutreach/mail/blob/5380819cf0e71ce19a2adea6740277db71b2b446/spec/mail/fields/references_field_spec.rb#L40) - they know it should have `\r\n` when it's encoded, but they don't parse it that way ...weird.

Reply:
Interesting, the test I added definitely fails. That's pretty weird though.